src/Compilers/CSharp/Portable/Symbols/Source/SourceTypeParameterSymbol.cs

```diff
@@ -171,12 +177,15 @@ internal override CustomAttributesBag<CSharpAttributeData> GetAttributesBag()
                 var sourceMethod = this.ContainingSymbol as SourceMemberMethodSymbol;
                 if ((object)sourceMethod == null || (object)sourceMethod.SourcePartialDefinition == null)
                 {
-                    lazyAttributesStored = LoadAndValidateAttributes(OneOrMany.Create(this.MergedAttributeDeclarationSyntaxLists), ref _lazyCustomAttributesBag);
+                    lazyAttributesStored = LoadAndValidateAttributes(
```

Comment:
Does that mean there's a race condition where the diagnostics are either added to a bag passed as an argument or to the declaration diagnostics, depending on which call is first? Is there a reason to pass in a bag rather than adding to declaration diagnostics always?